ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -2907,7 +2907,11 @@ private void SetProperties(SpreadsheetDocument dSpreadsheet)
                 xlStyle.NumberFormat = xlNumberFormat;
             }
 
-            xlStylized.InnerStyle = new XLStyle(xlStylized, xlStyle);
+            //When loading columns we must propagate style to each column but not deeper. In other cases we do not propagate at all.
+            if (xlStylized is IXLColumns columns)
+                columns.Cast<XLColumn>().ForEach(col => col.InnerStyle = new XLStyle(col, xlStyle));
+            else
+                xlStylized.InnerStyle = new XLStyle(xlStylized, xlStyle);
```

Comment:
Does this logic not hold also for `IXLRows` and may other similar types?

Reply:
Column and row styles are stored differently in Excel. Say, if we have a range of columns with the same style they will be loaded simultaneously (thus we'll get `XLColumns` here):
![image](https://user-images.githubusercontent.com/19576939/53088507-610ed400-3523-11e9-9f36-006cbae37cd2.png)
Rows are always stored separately (at least, I could not manage to generate a file where they are combined) so we will not get `XLRows`:
![image](https://user-images.githubusercontent.com/19576939/53088545-83085680-3523-11e9-9ff4-7aec2d9a76dc.png)

Probably we can still add similar logic for rows as well - just in case...